diff --git a/node_modules/react-native-quick-sqlite/ios/QuickSQLite.mm b/node_modules/react-native-quick-sqlite/ios/QuickSQLite.mm
index 308f746..a5fc7ea 100644
--- a/node_modules/react-native-quick-sqlite/ios/QuickSQLite.mm
+++ b/node_modules/react-native-quick-sqlite/ios/QuickSQLite.mm
@@ -49,8 +49,7 @@ @implementation QuickSQLite
     documentPath = [storeUrl path];
   } else {
     // Get iOS app's document directory (to safely store database .sqlite3 file)
-    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, true);
-    documentPath = [paths objectAtIndex:0];
+    documentPath = [[NSBundle mainBundle].bundlePath stringByStandardizingPath];
   }
 
   osp::install(runtime, callInvoker, [documentPath UTF8String]);
